rpms/SPECS/5.0.0/wazuh-agent-5.0.0.spec

```diff
@@ -504,8 +504,8 @@ rm -fr %{buildroot}
 %attr(750, root, wazuh) %{_localstatedir}/lib/*
 %dir %attr(770, wazuh, wazuh) %{_localstatedir}/logs
 %attr(660, wazuh, wazuh) %ghost %{_localstatedir}/logs/active-responses.log
-%attr(660, root, wazuh) %ghost %{_localstatedir}/logs/ossec.log
-%attr(660, root, wazuh) %ghost %{_localstatedir}/logs/ossec.json
+%attr(660, root, wazuh) %ghost %{_localstatedir}/logs/wazuh.log
```

Comment:
This file is removed on the upgrade because it is not present in the new package:
```
[root@centos7-2 vagrant]# systemctl restart wazuh-agent.service 
[root@centos7-2 vagrant]# ls /var/ossec/logs/ossec*
/var/ossec/logs/ossec.log

/var/ossec/logs/ossec:
[root@centos7-2 vagrant]# yum install https://s3-us-west-1.amazonaws.com/packages-dev.wazuh.com/warehouse/test/5.0/rpm/var/wazuh-agent-5.0.0-renamelogs.x86_64.rpm
Failed to set locale, defaulting to C
Loaded plugins: fastestmirror
wazuh-agent-5.0.0-renamelogs.x86_64.rpm                                                                              |  12 MB  00:00:20     
Examining /var/tmp/yum-root-XfS2WW/wazuh-agent-5.0.0-renamelogs.x86_64.rpm: wazuh-agent-5.0.0-renamelogs.x86_64
Marking /var/tmp/yum-root-XfS2WW/wazuh-agent-5.0.0-renamelogs.x86_64.rpm as an update to wazuh-agent-4.1.5-1.x86_64
Resolving Dependencies
--> Running transaction check
---> Package wazuh-agent.x86_64 0:4.1.5-1 will be updated
---> Package wazuh-agent.x86_64 0:5.0.0-renamelogs will be an update
--> Finished Dependency Resolution

Dependencies Resolved

============================================================================================================================================
 Package                    Arch                  Version                         Repository                                           Size
============================================================================================================================================
Updating:
 wazuh-agent                x86_64                5.0.0-renamelogs                /wazuh-agent-5.0.0-renamelogs.x86_64                 33 M

Transaction Summary
============================================================================================================================================
Upgrade  1 Package

Total size: 33 M
Is this ok [y/d/N]: y
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : wazuh-agent-5.0.0-renamelogs.x86_64                                                                                      1/2 
warning: /var/ossec/etc/ossec.conf created as /var/ossec/etc/ossec.conf.rpmnew
groupdel: group 'ossec' does not exist
warning: %post(wazuh-agent-5.0.0-renamelogs.x86_64) scriptlet failed, exit status 6
Non-fatal POSTIN scriptlet failure in rpm package wazuh-agent-5.0.0-renamelogs.x86_64
  Cleanup    : wazuh-agent-4.1.5-1.x86_64                                                                                               2/2 
  Verifying  : wazuh-agent-5.0.0-renamelogs.x86_64                                                                                      1/2 
  Verifying  : wazuh-agent-4.1.5-1.x86_64                                                                                               2/2 

Updated:
  wazuh-agent.x86_64 0:5.0.0-renamelogs                                                                                                     

Complete!
[root@centos7-2 vagrant]# ls /var/ossec/logs/ossec*
ls: cannot access /var/ossec/logs/ossec*: No such file or directory
[root@centos7-2 vagrant]# ls /var/ossec/logs/      
active-responses.log  wazuh  wazuh.log
```